submissions/codedentwickler/Week1/Fizzbuzz.playground/Contents.swift

```diff
@@ -1,21 +1,21 @@
-//: Playground - noun: a place where people can play
-
-import Foundation
-
-func fizzbuzz(_ i: Int) -> String {
-    let result = (i % 3, i % 5)
-    switch result {
-        case (0, 0):
-            return "FizzBuzz"
-        case (0, _):
-            return "Fizz"
-        case (_, 0):
-            return "Buzz"
-        default:
-            return String(i)
-    }
-}
-
-for number in 1...100 {
-    print(fizzbuzz(number))
-}
+//: Playground - noun: a place where people can play
+
+import Foundation
+
+func fizzbuzz(_ i: Int) -> String {
+    let result = (i % 3, i % 5)
+    switch result {
+        case (0, 0):
+            return "FizzBuzz"

```

Comment:
Ensure these changes are not showing on your PR. This is not your submission folder.